monster-cards/src/stores/favoriteStore.ts

```diff
@@ -0,0 +1,37 @@
+import createMemoryStore, { MemoryStore } from 'dojo-stores/createMemoryStore';
+import { favCard } from './../actions/widgetStoreActions';
+
+export interface Card {
+	id: string;
+	cardType: 'momentum' | 'mayhem';
+	name: string;
+	tagline: string;
+	description: string;
+	score: number;
+	cardImage: string;
+	favouriteCount: number;
+}
+
+export type Store = MemoryStore<Card>;
+
+const favCardStore: Store = createMemoryStore<Card>({
+	data: [
+	]
+});
+
+export interface ChangeRecord {
+	beforeAll: Card[];
+	afterAll: Card[];
+	deletes: string[];
+	puts: Card[];
+}
+
+export function bindActions() {
+	return favCardStore.observe().subscribe((options: any) => {
```

Comment:
this seems odd... why not `options: ChangeRecord`?  Why the copy to another var and then do nothing with it?

If `options` can't be coerced to `ChangeRecord` in the parameters for some reason, why not:

```
return facCard.do(<ChangeRecord> options);
```

I just don't think it adds anything to the clarity of the code.